api/institutions/serializers.py

```diff
@@ -191,3 +191,39 @@ def get_absolute_url(self, obj):
                 'version': 'v2',
             },
         )
+
+
+class InstitutionUserMetricsSerializer(JSONAPISerializer):
+
+    class Meta:
+        type_ = 'institution-users'
+
+    id = IDField(read_only=True)
+    user_id = IDField(read_only=True)
+    user = ser.CharField(read_only=True)
+    public_projects = ser.IntegerField(read_only=True)
+    private_projects = ser.IntegerField(read_only=True)
+    department = ser.CharField(read_only=True)
+
+    users = RelationshipField(
```

Comment:
I think this should just be `user` rather than `users` since it only returns one.